src/graph/DeleteVerticesExecutor.cpp

```diff
@@ -52,8 +53,44 @@ void DeleteVerticesExecutor::execute() {
         vids_.emplace_back(vid);
     }
 
-    // TODO(zlcook) Get edgeKeys of a vertex by Go
-    auto future = ectx()->getStorageClient()->getEdgeKeys(space_, vids_);
+    auto edgeAllStatus = ectx()->schemaManager()->getAllEdge(space_);
+    if (!edgeAllStatus.ok()) {
+        doError(edgeAllStatus.status());
+        return;
+    }
+
+    std::vector<EdgeType> edgeTypes;
+    std::vector<std::string> columns = {_DST, _RANK};
+    std::vector<nebula::storage::cpp2::PropDef> props;
+    for (auto &e : edgeAllStatus.value()) {
+        auto edgeStatus = ectx()->schemaManager()->toEdgeType(space_, e);
+        if (!edgeStatus.ok()) {
+            LOG(ERROR) << "Can't get all of the edge types";
```

Comment:
Is better  "Can't get edge type, edge : " << e;
It's no big deal,  LGTM.

Reply:
I think it's ok ~